src/Group.php

```diff
@@ -31,6 +31,25 @@ public function __construct(?string $prefix = null, ?callable $callback = null,
         return $factory($prefix, $routes);
     }
 
+    final public function setContainer(ContainerInterface $container): Group
+    {
+        $group = clone $this;
```

Comment:
Why it need? Why we can't apply new container value to `$this`?

Reply:
Road runner

Reply:
What do you mean? Can you explain more further?

Reply:
With RR we can set another|empty container dynamically and it will be used in the next request.

Reply:
I don't know case which will use this "feature". All routes will be cached and then dynamical configuring is not be an actual.
Also I don't see tests for this feature.

Reply:
It is a bug, not the feature

Reply:
> Also I don't see tests for this feature.
This requires an integration test.